docs/TempUG.md

```diff
@@ -103,15 +103,1078 @@ This project is based on the [AddressBook-Level3 project](https://se-education.o
+```
+This block of code is for success outputs
+```
+Failure outputs:
+* Input: `invalid command code here`
+* Explanation and solution here, this is because the flag has an incorrect value, bla bla bla
+```
+Invalid command with the error message here
+```
+
+* Input: `invalid command code here`
+* Explanation and solution here, this is because the flag has an incorrect value, bla bla bla
+```
+Invalid command with the error message here
+```
+
+<br>
+
+### Adding a student : `addPerson`
+
+About the feature
+
+Format: `command COMPULSORY [optional]`
+* Format info 1
+* Format info 2
+
+<box type="tip" seamless>
+
+**Tips:**
+- Tip 1
+- Tip 2
+
+</box>
+
+Example usages:
+* `some code here`
+* `another code here`
+
+Success outputs:
+* Input: `code with compulsory parameters`
+* Input: `code with compulsory and optional parameters`
+```
+This block of code is for success outputs
+```
+Failure outputs:
+* Input: `invalid command code here`
+* Explanation and solution here, this is because the flag has an incorrect value, bla bla bla
+```
+Invalid command with the error message here
+```
+
+* Input: `invalid command code here`
+* Explanation and solution here, this is because the flag has an incorrect value, bla bla bla
+```
+Invalid command with the error message here
+```
+
+<br>
+
+### Adding a Lesson : `addLesson`
+
+About the feature
+
+Format: `command COMPULSORY [optional]`
+* Format info 1
+* Format info 2
+
+<box type="tip" seamless>
+
+**Tips:**
+- Tip 1
+- Tip 2
+
+</box>
+
+Example usages:
+* `some code here`
+* `another code here`
+
+Success outputs:
+* Input: `code with compulsory parameters`
+* Input: `code with compulsory and optional parameters`
+```
+This block of code is for success outputs
+```
+Failure outputs:
+* Input: `invalid command code here`
+* Explanation and solution here, this is because the flag has an incorrect value, bla bla bla
+```
+Invalid command with the error message here
+```
+
+* Input: `invalid command code here`
+* Explanation and solution here, this is because the flag has an incorrect value, bla bla bla
+```
+Invalid command with the error message here
+```
+
+<br>
+
+### Adding a task: `addTask`
+
+About the feature
+
+Format: `command COMPULSORY [optional]`
+* Format info 1
+* Format info 2
+
+<box type="tip" seamless>
+
+**Tips:**
+- Tip 1
+- Tip 2
+
+</box>
+
+Example usages:
+* `some code here`
+* `another code here`
+
+Success outputs:
+* Input: `code with compulsory parameters`
+* Input: `code with compulsory and optional parameters`
+```
+This block of code is for success outputs
+```
+Failure outputs:
+* Input: `invalid command code here`
+* Explanation and solution here, this is because the flag has an incorrect value, bla bla bla
+```
+Invalid command with the error message here
+```
+
+* Input: `invalid command code here`
+* Explanation and solution here, this is because the flag has an incorrect value, bla bla bla
+```
+Invalid command with the error message here
+```
+
```

Comment:
i rmb we discussed to lump them as a "add" feature
then within that we split into addPerson, addLesson, addTask

rather than taking each as its own feature